test/unit/client.test.ts

```diff
@@ -749,6 +749,110 @@ describe('Client', () => {
     });
   });
 
+  describe('updateDefaultResolution()', () => {
+    it('should set the default video resolution to the proper value', () => {
+      sdk = constructSdk();
+      sdk.updateDefaultResolution({width: 1920, height: 1080}, false);
+      expect(sdk._config.defaults.videoResolution).toStrictEqual({width: 1920, height: 1080});
+
+      sdk.updateDefaultResolution(undefined, false);
+      expect(sdk._config.defaults.videoResolution).toBeUndefined();
+    })
+
+    it('will attempt to update the videos resolution to the requested value; resolution is defined', async () => {
+      sdk = constructSdk();
+      const stream = new MockStream() as any as MediaStream;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9629161)
```suggestion
      const stream = new MockStream() as unknown as MediaStream;
```